extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.json

```diff
@@ -0,0 +1,172 @@
+            "journey:stepStatus":{
+              "title":"stepStatus",
+              "description":"Status of the journey step, representing in which shape the step is, when its processing has been done",
+              "type":"string"
+            },
+            "journey:journeyId":{
+              "title":"journeyId",
+              "description":"ID of the journey",
+              "type":"string"
+            },
+            "journey:journeyVersionId":{
+              "title":"journeyVersionId",
+              "description":"ID of the journey version. This id represents the identity reference to the journey, in case of the journeyStepEvent",
+              "type":"string"
+            },
+            "journey:journeyVersionName":{
+              "title":"journeyVersionName",
+              "description":"Name of the journey version",
+              "type":"string"
+            },
+            "journey:journeyVersion":{
+              "title":"journeyVersion",
+              "description":"Version of the journey version",
+              "type":"string"
+            },
+            "journey:instanceId":{
+              "title":"instanceId",
+              "description":"Internal ID of the journey instance",
+              "type":"string"
+            },
+            "journey:externalKey":{
+              "title":"externalKey",
+              "description":"External key extracted from the event to process it",
+              "type":"string"
+            },
+            "journey:parentStepId":{
+              "title":"parentStepId",
+              "description":"Step ID of the parent of the current processed step in the instance",
+              "type":"string"
+            },
+            "journey:parentStepName":{
+              "title":"parentStepName",
+              "description":"Step name of the parent of the current step.",
+              "type":"string"
+            },
+            "journey:parentTransitionId":{
+              "title":"parentTransitionId",
+              "description":"Id of the transition which has brought the instance to the processed step",
+              "type":"string"
+            },
+            "journey:parentTransitionName":{
+              "title":"parentTransitionName",
+              "description":"Name of the transition which has brought the instance to the processed step",
+              "type":"string"
+            },
+            "journey:inTest":{
+              "title":"inTest",
+              "description":"Whether this journey is in test mode or not",
+              "type":"boolean"
+            },
+            "journey:processingTime":{
+              "title":"processingTime",
+              "description":"total time in millis from the instance step entrance to the end of the processing",
+              "type":"integer"
+            },
+            "journey:instanceType":{
+              "title":"instanceType",
+              "description":"indicate the instance type, if it is batch or unitary",
```

Comment:
If there is a limited set of values - make it an enum

Reply:
there can be different instance types batch, unitary and later some more as well so we are using it as a string.